src/ctia/stores/es/store.clj

```diff
@@ -40,6 +40,10 @@
              [verdict :as ve]]
             [schema.core :as s]))
 
+(defn delete-state-indexes [{:keys [conn index config]}]
+  (when conn
+    (es-index/delete! conn (str index "*"))))
```

Comment:
From the ES documentation:
https://www.elastic.co/guide/en/elasticsearch/reference/5.0/indices-delete-index.html

> In order to disable allowing to delete indices via wildcards or _all, set action.destructive_requires_name setting in the config to true. This setting can also be changed via the cluster update settings api.

Do we need to change the config for our local ES before this ES purge task will work correctly?